### PR TITLE
feat: added 404 page and its resources

### LINKED ISSUE #4: Create 404 page error
Use existent layout:
https://gist.github.com/RenatoSiqueira/e6c780e61ec20a762f35eeeb49246387

### PR DESCRIPTION
**Pull request by empiresdev**

### CODE REVIEW

File: pages/404.tsx [hunk 1]
```diff
@@ -0,0 +1,28 @@
+import type { NextPage } from 'next'
+import Image from 'next/image'
+import Link from 'next/link'
+import error404 from '../public/404/error-404.svg'
```
Comment (RenatoSiqueira) [inline pages/404.tsx]: Replace this import to use recommended NextJS image method.

File: styles/globals.css [hunk 2]
```diff
@@ -1,3 +1,7 @@
 @tailwind base;
 @tailwind components;
 @tailwind utilities;
+
+.bg-404{
+    background-image: url("../public/404/bg.png");
```
Comment (RenatoSiqueira) [inline styles/globals.css]: Replace this import to use recommended NextJS method.

**Review by RenatoSiqueira — CHANGES_REQUESTED** [review]

File: pages/404.tsx (hunk 1 above)
Comment (empiresdev) [reply pages/404.tsx]: @RenatoSiqueira I thought this was the recommended NextJS method: https://nextjs.org/docs/basic-features/image-optimization#local-images

Can you please explain in more details what would be done to solve this issue?

**Comment by empiresdev** [comment]
@RenatoSiqueira I made changes as requested and commited them to feature/404 branch. Could someone review this PR?

**Review by RenatoSiqueira — APPROVED** [review]

**Pull request merged**